build.sbt

```
@@ -238,6 +238,30 @@ lazy val semconv = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     name := "otel4s-semconv"
   )
 
+lazy val `sdk-common` = crossProject(JVMPlatform, JSPlatform, NativePlatform)
+  .crossType(CrossType.Pure)
+  .in(file("sdk/common"))
+  .dependsOn(`core-common`)
+  .settings(
+    name := "otel4s-sdk-common",
+    libraryDependencies ++= Seq(
+      "org.typelevel" %%% "cats-effect-kernel" % CatsEffectVersion,
+      "org.typelevel" %%% "cats-mtl" % CatsMtlVersion,
+      "org.scalameta" %%% "munit" % MUnitVersion % Test
+    ),
+  )
+
+lazy val `sdk-trace` = crossProject(JVMPlatform, JSPlatform, NativePlatform)
+  .crossType(CrossType.Pure)
+  .in(file("sdk/trace"))
+  .dependsOn(`sdk-common`, `core-trace`)
+  .settings(
+    name := "otel4s-sdk-trace",
+    libraryDependencies ++= Seq(
+      "org.typelevel" %%% "cats-effect" % CatsEffectVersion,
+    ),
+  )
```

Probably we can add `munitDependencies` to both modules. You are planning to write tests for tracing right 😜 

```suggestion
lazy val `sdk-common` = crossProject(JVMPlatform, JSPlatform, NativePlatform)
  .crossType(CrossType.Pure)
  .in(file("sdk/common"))
  .dependsOn(`core-common`)
  .settings(
    name := "otel4s-sdk-common",
    libraryDependencies ++= Seq(
      "org.typelevel" %%% "cats-effect-kernel" % CatsEffectVersion,
      "org.typelevel" %%% "cats-mtl" % CatsMtlVersion
    ),
  )
  .settings(munitDependencies)

lazy val `sdk-trace` = crossProject(JVMPlatform, JSPlatform, NativePlatform)
  .crossType(CrossType.Pure)
  .in(file("sdk/trace"))
  .dependsOn(`sdk-common`, `core-trace`)
  .settings(
    name := "otel4s-sdk-trace",
    libraryDependencies ++= Seq(
      "org.typelevel" %%% "cats-effect" % CatsEffectVersion,
    ),
  )
  .settings(munitDependencies)
```

---

Oops, just saw this. Will revise to add :)